src/app/providers/AlertProvider.tsx

```diff
@@ -22,7 +23,7 @@ export const AlertProvider: FC<Props> = ({ children }) => {
 export const useAlertContext = () => {
   const context = useContext(AlertContext)
   if (!context) {
-    throw new Error('useAlertContext must be used within an AlertProvider')
+    throw new NoContextProviderError('useAlertContext', 'AlertProvider')
```

Comment:
Maybe we should also do this in the `AllocationsContext`

Reply:
I wasn't going to go through all contexts, I only replaced the errors that follow the same pattern.